lgtm.yml

```diff
@@ -0,0 +1,7 @@
+extraction:
+  go:
+    before_index:
+    - "sed -i 's=$(TOP_DIR)/sonic-mgmt-common=/opt/src/sonic-mgmt-common=g' Makefile"
+    - "sed -i 's@replace github.com/Azure/sonic-mgmt-common => ../sonic-mgmt-common@replace\
+      \ github.com/Azure/sonic-mgmt-common => /opt/src/sonic-mgmt-common@g' go.mod"

```

Comment:
Is it same as just deleting this line?

Reply:
The changes to go.mod are needed to simply tell LGTM that the sonic-mgmt-common directory is located in the current director (./sonic-mgmt-common) and not in the parent directory (../sonic-mgmt-common)